ckan/model/meta.py

```diff
@@ -79,9 +78,9 @@ def ckan_after_rollback(session: Any):
         del session._object_cache
 
 
-#mapper = Session.mapper
 mapper = orm.mapper
 
 # Global metadata. If you have multiple databases with overlapping table
 # names, you'll need a metadata for each database
```

Comment:
```suggestion
```


I think it is safe to remove this comment right? Are we handling multiple databases?